pkg/drivers/vfkit/driver_darwin.go

```diff
@@ -488,36 +486,10 @@ func (d *Driver) sendSignal(s syscall.Signal) error {
 	return proc.SendSignal(s)
 }
 
-var (
-	once              sync.Once
-	macCurrentVersion string
-	errGettingVersion error
-)
-
 func (d *Driver) supportsVirtiofs() bool {
-	supportsVirtioFS, err := macosAtLeast("12.0.0")
+	supportsVirtioFS, err := darwin.AtLeast("12.0.0")
```

Comment:
Not asking for changes, but one could argue this should be `darwin.AtLeast("21.0.0")`, and that `AtLeast` needs to be rewritten (darwin and macos versions are different things https://en.wikipedia.org/wiki/Darwin_(operating_system) )